### PR TITLE
add jira workflow

### PR DESCRIPTION
**Pull request by carlosaicrag**
adds jira workflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
 
 ## Workflow
 * So you want to contribute to cohort resources? you've come to the right spot
+### Jira Workflow 
+* [Jira Link](https://appacademyio.atlassian.net/secure/RapidBoard.jspa?rapidView=86&projectKey=IN&selectedIssue=IN-91)
+* This is a slightly modified outline of the workflow for the curriculumn team
```
Comment (mkayys) [inline README.md:112]: curriculum --> cohort resources

File: README.md [hunk 2]
```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
...
+#### submitting a ticket
+* Submitting a ticket is quite easy. First create a new task on the Curriculum Issues Backlog:
```
Comment (mkayys) [inline README.md:114]: Curriculum --> Instructional?

File: README.md [hunk 3]
```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
...
+    * This is the 'at a glance' view of the issue being created. Please include the course where the issue is occurring as well as a terse description (~15 words or less) of what's going wrong.
+
+    * Example: [CR] add study guide for active record
+
+* Description
+    * Here is your chance to explain the issue as best you can in as much detail as you can muster and give us everything we need to fix the issue. A few things MUST be present:
+
+    1. a link to the place where the issue is happening or where you are adding something
+    2. a description of what you observe and why this conflicts with what you think it should be
+    priority, in your opinion (described below)
```
Comment (mkayys) [inline README.md:135]: I think we're missing a **3.** here for the priority

File: README.md [hunk 4]
```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
...
+* What to expect
+    * We on the cohort resources team meet regularly and prioritize issues to fix. We will evaluate them based on urgency and our availability.
+    * When we do go over the issue, we might ask you questions about your issue. 
+
+
+### git Workflow 
+* Before contributing please make sure that you follow the steps above to submit an issue and assign yourself to it once it has been placed in the selected for development section
 * First of all, NEVER push to master.
 * Demos during lecture will follow the instructions below [Lecture Demos](#lecture-demos)
 * Any contributions to the master branch will follow the following workflow:
```
Comment (mkayys) [inline README.md:156]: ends with **:** but doesn't list anything afterwards

File: README.md [hunk 5]
```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
...
+* Description
+    * Here is your chance to explain the issue as best you can in as much detail as you can muster and give us everything we need to fix the issue. A few things MUST be present:
+
+    1. a link to the place where the issue is happening or where you are adding something
+    2. a description of what you observe and why this conflicts with what you think it should be
+    priority, in your opinion (described below)
+        * To make our life even easier please provide a screenshot if applicable or even a suggestion for how the problem should be solved.
+
+* Estimating priority
+    * Please set priority in line with the following guidelines.
```
Comment (mkayys) [inline README.md:139]: small thing, possibly personal preference, but ending this line with a **:** (colon)?

File: README.md (hunk 4 above)
Comment (carlosaicrag) [reply README.md:156]: the list already exists in the main readme. the green is why ive added and everything else is stuff that is already there

File: README.md (hunk 3 above)
Comment (carlosaicrag) [reply README.md:135]: good catch!

File: README.md (hunk 2 above)
Comment (carlosaicrag) [reply README.md:114]: fixed

File: README.md (hunk 1 above)
Comment (carlosaicrag) [reply README.md:112]: I just went ahead and removed this sentence

File: README.md (hunk 4 above)
Comment (mkayys) [reply README.md:156]: sweet! Just couldn't see anything in this preview

**Pull request merged**